### PR TITLE
docs: Add a new proposal: Shorten the On-Chain voting period

### PR DESCRIPTION
**Pull request by gyuguen**

### CODE REVIEW

**Comment by gyuguen** [comment]
I am going to test on devnet.

File: proposals/2021-11-voting_period/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+We would like to change parameter as follows.
+- The voting period for the proposal will be changed from 14 days to 3 days.
+
+
+### Background
+
+The voting period for governance in Panacea is 14 days, which is enough time to discuss the proposal.<br/> 
+However, this period is too long to keep up with this fast-paced crypto ecosystem.<br/>
+As an example, simple changes in specific parameters for IBC transmission took 14 days, which takes a lot of time to link to Osmosis.
```
Comment (youngjoon-lee) [inline proposals/2021-11-voting_period/README.md]: For more clarity,
```suggestion
As an example, simple changes in specific parameters for IBC transfer took 14 days, which which made it a long time to integrate Panacea with other IBC-based services such as Osmosis.
```

**Review by youngjoon-lee — APPROVED** [review]
LGTM

**Comment by gyuguen** [comment]
I tested in devnet. 
스크린샷 2021-11-24 오후 2 17 47

**Pull request merged**